Makefile

```diff
@@ -128,6 +139,17 @@ dumpvar:
 	fi
 	@echo "$(VAR) is '$($(VAR))'"
 
+# eslint ftw
+ESLINT = ./node_modules/.bin/eslint
+$(ESLINT):
+	$(RUN_NPM_INSTALL)
+
+check:: check-eslint
+
+.PHONY: check-eslint
+check-eslint: $(ESLINT)
+	@./node_modules/.bin/eslint ./
```

Comment:
On 16 December 2015 at 11:08, Julien Gilli <notifications@github.com> wrote:
> Is it possible to add a "npm script" in package.json named "eslint" and run npm run-script eslint here instead? That would remove the need for hardcoding paths to ./node_modules/.bin/eslint

Is the "node_modules/.bin/<program>" contract likely to be broken at
some point?  It seems better to just run the program in the Makefile
than to indirect through npm.

-- 
Joshua M. Clulow
Software Engineer @ Joyent
mail: jmc@joyent.com